tests/system/alerts/test_alerts.py

```diff
@@ -175,6 +175,11 @@ def _generate_events(
             system_performance_example,
         ]
 
+    @staticmethod
+    def _normalize_alert_name(alert_kind):
+        sanitized_name = alert_kind.value.replace("_", "-")
+        return sanitized_name
```

Comment:
you can use `mlrun.utils.helpers.normalize_name` instead of creating a function here